documentation/modules/exploit/multi/http/kong_gateway_admin_api_rce.md

```diff
@@ -0,0 +1,88 @@
+## Vulnerable Application
+
+Kong Gateway claims to be the "world’s most popular open source API gateway". It allows API operators to add features, such as
+Authentication, Traffic Control, Analytics, Transformations, Logging and even serverless functions to existing APIs.
+
+It is open-source, can be downloaded from [konghq.com](https://konghq.com/get-started/#install) and runs on Linux or macOS. Many officially
+supported packages are available, for example from repositories, DockerHub or AMI images for AWS.
+
+This exploit module uses the [Admin API](https://docs.konghq.com/2.1.x/admin-api/) to create a route, then assign a the
+[pre-function](https://docs.konghq.com/hub/kong-inc/serverless-functions/) serverless plugin to that route. The plugin runs Lua code and is
+used to run a system command using `os.execute()`. After execution, the route is then deleted, which also results in the plugin associated
+with the route being deleted.
+
+The Admin API, by default (since version 0.12.0) is bound to localhost and therefore shouldn't be available externally. It is, however,
+possible to remove that restriction in the configuration. [The documentation](https://docs.konghq.com/2.1.x/secure-admin-api/) states that
+"*Kong’s routing design allows it to serve as a proxy for the Admin API itself. In this manner, Kong itself can be used to provide
+fine-grained access control to the Admin API.*"
+
```

Comment:
since this isn't a specific version that is vulnerable, I'd add a section:

```
Successfully tested against:
* Version xyz
```

Reply:
Added